miniapps/dpg/makefile

```diff
@@ -12,6 +12,7 @@
 # Use the MFEM build directory
 MFEM_DIR ?= ../..
 MFEM_BUILD_DIR ?= ../..
+MFEM_INSTALL_DIR ?= ../../mfem
 SRC = $(if $(MFEM_DIR:../..=),$(MFEM_DIR)/miniapps/dpg/,)
 CONFIG_MK = $(MFEM_BUILD_DIR)/config/config.mk
```

Comment:
Update the definition of `CONFIG_MK` here as well.

Reply:
My bad. Thanks for spotting this!